ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2128,7 +2104,27 @@ def test033_Duplicate_main_order_and_cahange_materiel_type(self):
         self.assertEqual(suborder_data['Article Name'].replace("'", ""), selected_test_plan['article'][0])
         self.assertEqual(suborder_data['Test Units'], test_unit)
         self.assertEqual(suborder_data['Test Plans'], selected_test_plan['testPlanName'])
-        
+
+    def test035_archived_test_unit_shoudnt_display_in_the_order_drop_down_list(self):
+        """
+        Orders: Archived Test unit: Archive Approach: Archived test units shouldn't appear in orders in the drop down list
+        LIMS-3710
+        :return:
+        """
+        testunits, payload = self.test_unit_api.get_all_test_units(limit=20, deleted=1)
+        random_row = random.choice(testunits['testUnits'])
+
+        self.order_page.get_orders_page()
```

Comment:
Remove this line

Reply:
@dianaeldemerdash

Reply:
Done

Reply:
@dianaeldemerdash seems u didn't remove it yet